spec/lib/gds_api/whitehall_export_spec.rb

```diff
@@ -2,22 +2,21 @@
 
 RSpec.describe GdsApi::WhitehallExport do
   describe "document_list" do
-    let(:whitehall_adapter) { GdsApi::WhitehallExport.new(Plek.find("whitehall-admin")) }
     let(:whitehall_host) { Plek.new.external_url_for("whitehall-admin") }
```

Comment:
I think we can get rid of this now that we're calling `#stub_whitehall_api_has_document_index`.

Reply:
Yes, this isn't needed, so I've removed it